keras/legacy/interfaces.py

```diff
@@ -85,6 +92,15 @@ def raise_duplicate_arg_error(old_arg, new_arg):
     allowed_positional_args=['alpha_initializer'],
     conversions=[('init', 'alpha_initializer')])
 
+
 legacy_gaussiannoise_support = generate_legacy_interface(
     allowed_positional_args=['stddev'],
     conversions=[('sigma', 'stddev')])
+
+legacy_pooling2d_support = generate_legacy_interface(
+    allowed_positional_args=['pool_size', 'strides', 'padding'],
+    conversions=[('pool_length', 'pool_size'),
+                 ('stride', 'strides'),
```

Comment:
`stride` and `pool_length` are not arguments of pooling 2d layers (that was pooling 1d layers).